src/core/zowe/core_for_zowe_sdk/profile_manager.py

```diff
@@ -299,19 +251,172 @@ def load_credentials(self) -> dict:
+
+        self.global_config = ConfigFile(type=TEAM_CONFIG, name=GLOBAL_CONFIG_NAME)
+        try:
+            self.global_config.location = GLOBAl_CONFIG_LOCATION
+        except Exception:
+            warnings.warn("Could not find Global Config Directory, please provide one.")
+
+        self.global_user_config = ConfigFile(type=USER_CONFIG, name=GLOBAL_CONFIG_NAME)
+        try:
+            self.global_user_config.location = GLOBAl_CONFIG_LOCATION
+        except Exception:
+            warnings.warn(
+                "Could not find Global User Config Directory, please provide one."
+            )
+
+    @property
+    def config_appname(self) -> str:
+        """Returns the app name"""
+        return self._appname
+
+    @property
+    def config_dir(self) -> Union[str, None]:
+        """Returns the folder path to where the Zowe z/OSMF Team Project Config files are located."""
+        return self.project_config.location
+
+    @config_dir.setter
+    def config_dir(self, dirname: str) -> None:
+        """
+        Set directory/folder path to where Zowe z/OSMF Team Project Config files are located
+        """
+        self.project_config.location = dirname
+        self.project_user_config.location = dirname
+
+    @property
+    def user_config_dir(self) -> Union[str, None]:
+        """Returns the folder path to where the Zowe z/OSMF User Project Config files are located."""
+        return self.project_user_config.location
+
+    @user_config_dir.setter
+    def user_config_dir(self, dirname: str) -> None:
+        """Set directory/folder path to where Zowe z/OSMF User Project Config files are located"""
+        self.project_user_config.location = dirname
+
+    @property
+    def config_filename(self) -> str:
+        """Return the filename for Zowe z/OSMF Team Project Config"""
+        return self.project_config.filename
+
+    @property
+    def config_filepath(self) -> Union[str, None]:
+        """Get the full Zowe z/OSMF Team Project Config filepath"""
+        return self.project_config.filepath
+
+    def load(
+        self,
+        profile_name: Union[str, None] = None,
+        profile_type: Union[str, None] = None,
+    ) -> dict:
+        if profile_name is None and profile_type is None:
+            raise ProfileNotFound(
+                "Could not find profile as both profile_name and profile_type is not set."
+            )
+
+        service_profile: dict = {}
+        project_profile: dict = {}
+        global_profile: dict = {}
+        global_base_profile: dict = {}
+        project_profile_name: Union[str, None] = None
+        global_profile_name: Union[str, None] = None
+
+        # get Project Profile
+        try:
+            project_profile, project_profile_name = self.project_config.get_profile(
+                profile_name=profile_name, profile_type=profile_type
+            )
+        except Exception as exc:
+            warnings.warn(f"Could not load Project Config {self.project_config.name}")
```

Comment:
Thanks for replacing errors with warnings where the exception isn't fatal and it makes sense to try continuing 👍 

For debugging purposes it would be useful to include the exception message in the warning. Could you update this and all the "Could not load X Config" warnings below to include the exception? Something like this:
```
warnings.warn(f"Could not load Project Config {self.project_config.name}:\n{exc}")
```